esp/useful_scripts/migration_check.sh

```diff
@@ -3,7 +3,7 @@
 # Pipe the output of "manage.py migrate --list" to this script.
 # It will say whether there are unapplied django-south migrations.
 
-MIGRATIONS="`grep -v '(\*)' | grep -1 '( )'`"
```

Comment:
Ah. We might want that then, or at least `-B 1`, since most of the lines themselves just say "[ ] 0001_initial".